request_token/context_processors.py

```diff
@@ -0,0 +1,15 @@
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.functional import SimpleLazyObject
+
+
+def request_token(request):
```

Comment:
Context processor returns the full `jwt()` output - not the raw token object - as we want the middleware to handle the token the same way it handles a GET request.